tools/run_tests/performance/scenario_config.py

```diff
@@ -725,6 +726,133 @@ def scenarios(self):
     def __str__(self):
         return 'csharp'
 
+class DotnetLanguage(Language):
```

Comment:
The newly added grpc-dotnet scenarios are basically clones of the original CSharpLanguage scenarios.